### DISCUSSION TITLE
defineEventHandler and eventHandler which one is the alias?

### DISCUSSION THREAD

**Discussion by angelhdzmultimedia** [nuxt/nuxt#24397] [Questions]
![image](https://github.com/nuxt/nuxt/assets/10191085/bd83f444-bdd1-49d2-a724-f6aaa3a7df9b)

Hi, please help me understand this logic. 
Nuxt is built on top of unjs/h3 and unjs/nitro, so in unjs we have eventHandler, so isn't defineEventHandler the alias for eventHandler and not the opposite? Is this a typo?

Information will be appreciated.

**Comment by pi0** [comment]
They are identically the same & for consistency, i recommend to stick with longer version (`defineEventHandler`) -- docs also says same shorter is alias for lazy people :p

**Reply by angelhdzmultimedia** [reply]
😂 ha ha ha.

Yes I prefer the long name for consistency's sake. Thank you for your kindness, wish you lots of health and peace. 
Big fan of your work. 💚🔥😊🤝
